MapboxCoreNavigation/RouteController.swift

```diff
@@ -182,6 +182,10 @@ open class RouteController: NSObject {
         self.resumeNotifications()
         self.startEvents(route: route)
         self.resetSession()
+        
+        if !(route.routeOptions is NavigationRouteOptions) {
+            print("`Route` was created using `RouteOptions` and not `NavigationRouteOptions`. Although not required, this may lead to a suboptimal navigation experience.")
```

Comment:
Let’s spell out what the developer gains by using NavigationRouteOptions: congestion coloring along the route line, better ETAs, what else?